admin-dev/themes/new-theme/scss/pages/_modules.scss

```diff
@@ -528,9 +528,9 @@ form#importDropzone {
 
 .hook-panel {
   margin-bottom: 10px !important;
-  border: solid #E6E6E6 1px;
-  box-shadow: rgba(black,0.10) 0 -2px 0 inset;
-  border-radius: 3px;
+  // border: solid #E6E6E6 1px;
+  // box-shadow: rgba(black,0.10) 0 -2px 0 inset;
+  // border-radius: 3px;
```

Comment:
Please do not commit commented out lines. Either keep them or remove them.

Reply:
I removed  them